examples/disk/snapshot-restore/sts.yaml

```diff
@@ -1,46 +1,58 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: nginx
+  name: mysql
   labels:
-    app: nginx
+    app: mysql
 spec:
   ports:
-  - port: 80
-    name: web
+    - port: 3306
   clusterIP: None
   selector:
-    app: nginx
+    app: mysql
 ---
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: web
+  name: mysql
 spec:
   selector:
     matchLabels:
-      app: nginx
-  serviceName: "nginx"
+      app: mysql
+  serviceName: "mysql"
   replicas: 1
   template:
     metadata:
       labels:
-        app: nginx
+        app: mysql
     spec:
       containers:
-      - name: nginx
-        image: nginx
-        command: ["sh", "-c"]
-        args: ["sleep 10000"]
-        volumeMounts:
-        - name: disk-ssd
-          mountPath: /data
+        - name: mysql
+          image: mysql:8.0
```

Comment:
@vie-serendipity Please verify that this statefulset works in the k8s

Reply:
mysql:8.0 works fine.

Reply:
okay~